components/CreateTaskForm/CreateTaskForm.js

```diff
@@ -0,0 +1,105 @@
+import styled from "styled-components";
+import { useState } from "react";
+import { useEffect } from "react";
+
+const StyledCreateButton = styled.button`
+  border: 1px solid black;
+  border-radius: 50%;
+  background: grey;
+  text-decoration: none;
+  color: white;
+  font-weight: bold;
+  width: 70px;
+  height: 70px;
+  position: fixed;
+  bottom: 50px;
+  right: 50px;
+`;
+
+const StyledTaskForm = styled.form`
+  position: absolute;
+  top: 5%;
+  left: 5%;
+  background: #e0e1dd;
+  padding: 1.5rem 2rem;
+  display: flex;
+  flex-direction: column;
+  height: 90%;
+  width: 90%;
+  border: 1px solid #000;
+`;
+
+export default function CreateTaskForm({ onCreateTask }) {
+  const [createMode, setCreateMode] = useState(false);
+  const [today, setToday] = useState("");
+
+  useEffect(() => {
+    const currentDate = new Date();
+    const day = String(currentDate.getDate()).padStart(2, "0");
+    const month = String(currentDate.getMonth() + 1).padStart(2, "0");
+    const year = currentDate.getFullYear();
+
+    const formattedDate = `${year}-${month}-${day}`;
+    setToday(formattedDate);
+  }, []);
```

Comment:
This does work, but you should be able to achieve the identical result going with this approach:

```suggestion
const today = new Date().toISOString().split("T")[0];
```

By doing this, you can remove the state and no longer rely on the `useEffect`